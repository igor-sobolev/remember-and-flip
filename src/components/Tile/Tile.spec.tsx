import React from 'react';

import { Tile } from '.';
import { shallow, ShallowWrapper } from 'enzyme';

const givenProps = {
  size: 100,
  isFilled: true,
  isFlipped: true
};

describe('Given Tile component', () => {
  describe('when it is rendered', () => {
    let wrapper: ShallowWrapper;
    beforeEach(() => {
      wrapper = shallow(<Tile {...givenProps} />);
    });

    it('should match the snapshot', () => {
      expect(wrapper.getElement()).toMatchSnapshot();
    });
  });
});
