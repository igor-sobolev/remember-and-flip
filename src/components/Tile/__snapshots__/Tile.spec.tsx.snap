// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given Tile component when it is rendered should match the snapshot 1`] = `
<ReactCardFlip
  cardStyles={
    Object {
      "back": Object {},
      "front": Object {},
    }
  }
  cardZIndex="auto"
  containerStyle={Object {}}
  flipDirection="horizontal"
  flipSpeedBackToFront={0.6}
  flipSpeedFrontToBack={0.6}
  infinite={false}
  isFlipped={true}
>
  <TileCard
    className="tile"
    options={
      Object {
        "height": 96,
        "margin": 2,
        "width": 96,
      }
    }
  />
  <TileCard
    className="tile tile--filled"
    options={
      Object {
        "height": 96,
        "margin": 2,
        "width": 96,
      }
    }
  />
</ReactCardFlip>
`;
